controllers/projectreference_adapter.go

```diff
@@ -538,7 +529,7 @@ func (r *ReferenceAdapter) configureBillingAPI() error {
 		return err
 	}
 
-	if !util.Contains(enabledAPIs, "cloudbilling.googleapis.com") {
+	if !gcputil.Contains(enabledAPIs, "cloudbilling.googleapis.com") {
```

Comment:
wasn't is called it `util` in the other files?

Reply:
Right, I found a lot of pkg naming inconsistencies along the way and changed them to my best extent.
Changing this to `util`.